.eslintrc.js

```diff
@@ -262,17 +262,19 @@ module.exports = {
   'overrides': [
     {
       'files': [
-        'test/**/*.js',
-        'extensions/**/test/**/*.js',
-        'extensions/**/test-e2e/*.js',
         'ads/**/test/**/*.js',
+        'examples/visual-tests/**/*.js',
+        'extensions/**/test-e2e/*.js',
+        'extensions/**/test/**/*.js',
+        'test/**/*.js',
         'testing/**/*.js',
       ],
       'rules': {
         'require-jsdoc': 0,
         'local/always-call-chai-methods': 2,
         'local/no-bigint': 0,
         'local/no-dynamic-import': 0,
+        'local/no-for-of-statement': 0,
```

Comment:
`for...of` is fine to use on tests because they either run on node or they're transpiled down.